template/.github/workflows/on_pr_push_master.yml

```diff
@@ -1,14 +1,79 @@
-name: CI
-
+name: CI/CD
+env:
+  GITHUB_TOKEN: ${{ secrets.PAT_GITHUB_TOKEN }}
```

Comment:
Not necessary for template. Only needed in parent repo to push to template repo.
```suggestion
```

Reply:
Whoops, thanks!